src/components/PokemonCard.js

```diff
@@ -1,25 +1,35 @@
-import React from "react";
-import { Card } from "semantic-ui-react";
+import React, { useState } from "react"
+import { Card } from "semantic-ui-react"
 
-function PokemonCard() {
+function PokemonCard({ pokemon }) {
+  const { name, hp, sprites } = pokemon
+  const [sprite, setSprite] = useState(sprites.front)
+
+  function handleClick() {
+    if (sprite === sprites.front) {
+      setSprite(sprites.back)
+    } else if (sprite === sprites.back) {
+      setSprite(sprites.front)
+    }
+  }
   return (
     <Card>
       <div>
```

Comment:
A space here would be nice. 
```suggestion
  function handleClick() {
    if (sprite === sprites.front) {
      setSprite(sprites.back)
    } else if (sprite === sprites.back) {
      setSprite(sprites.front)
    }
  }
  
  return (
    <Card>
      <div>
```